projects/media-viewer/src/lib/toolbar/main-toolbar/main-toolbar.component.html

```diff
@@ -10,14 +10,15 @@
       <mv-tb-left-pane [currentPage]="currentPage"
                        [changePageByDelta]="changePageByDeltaEvent"
                        [setCurrentPage]="setCurrentPageEvent"
-                       [highlightMode]="toolbarButtons?.highlightMode"
                        [showSidebarToggleBtn]="toolbarButtons?.showSidebarToggleBtn"
                        [showSearchbarToggleBtn]="toolbarButtons?.showSearchbarToggleBtn"
                        [showNavigationBtns]="toolbarButtons?.showNavigationBtns"
                        [searchBarHidden]="toolbarButtons?.searchBarHidden"
                        [showHighlightBtn]="toolbarButtons?.showHighlightBtn"
-                       [drawMode]="toolbarButtons?.drawMode"
-                       [sidebarOpen]="toolbarButtons?.sidebarOpen"></mv-tb-left-pane>
+                       [sidebarOpen]="toolbarButtons?.sidebarOpen"
+                       [highlightMode]="false"
+                       [drawMode]="false"
```

Comment:
do we need to pass these through given that they're now coming from the toolbarEventsService

Reply:
@benouaer removed, can you recheck?